test/libsolidity/semanticTests/smoke_test.sol

```diff
@@ -1,8 +1,11 @@
 contract C {
     uint public state = 0;
-    constructor(uint _state) public {
+    constructor(uint _state) public payable {
         state = _state;
```

Comment:
Please add a check that the ether is actually sent.

Reply:
That's done in line 46: `// balance() -> 2`